frontend/src/component/playground/Playground/AdvancedPlaygroundResultsTable/combinationCounter.ts

```diff
@@ -0,0 +1,12 @@
+import { AdvancedPlaygroundFeatureSchema } from 'openapi';
+
+export const countCombinations = (
```

Comment:
Maybe backend should return it?

Reply:
(For future visibility; as mentioned in the slack discussion)

That's a good point! I put it in the frontend for now because that’s where we track events. Seemed sensible to do the calculations as close to where we track them as possible instead of sending them over the wire. That said, we could update the advanced playground response schema to include the count instead, but this seems like it would incur more complexity. I think the best reason for not doing it in the frontend is to reduce the amount of computation client side (and the bundle size, I suppose). That might be a worthwhile tradeoff, but how about we see whether it causes any performance problems first?